src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -10500,6 +10500,89 @@ .maxstack  1
       IL_0023:  ldloc.s    V_8
       IL_0025:  ret
     }
+");
+        }
+
+        [Fact]
+        public void AddUsing_AmbiguousCode()
+        {
+            var source0 = MarkedSource(@"
+using System.Threading;
+
+class C
+{
+    static void E() 
+    {
+        var t = new Timer(s => System.Console.WriteLine(s));
+    }
+}");
+            var source1 = MarkedSource(@"
+using System.Threading;
+using System.Timers;
+
+class C
+{
+    static void E() 
+    {
+        var t = new Timer(s => System.Console.WriteLine(s));
+    }
+    
+    static void G() 
+    {
+        System.Console.WriteLine(new TimersDescriptionAttribute(""""));
+    }
+}");
+
+            var compilation0 = CreateCompilation(source0.Tree, options: ComSafeDebugDll);
+            var compilation1 = compilation0.WithSource(source1.Tree);
+
+            var e0 = compilation0.GetMember<MethodSymbol>("C.E");
+            var e1 = compilation1.GetMember<MethodSymbol>("C.E");
+            var g1 = compilation1.GetMember<MethodSymbol>("C.G");
+
+            var v0 = CompileAndVerify(compilation0);
+            var md0 = ModuleMetadata.CreateFromImage(v0.EmittedAssemblyData);
+            var generation0 = EmitBaseline.CreateInitialBaseline(md0, v0.CreateSymReader().GetEncMethodDebugInfo);
+
+            // Pretend there was an update to C.E to ensure we haven't invalidated the test
+
+            var diffError = compilation1.EmitDifference(
+              generation0,
+              ImmutableArray.Create(
+                  SemanticEdit.Create(SemanticEditKind.Update, e0, e1, GetSyntaxMapFromMarkers(source0, source1), preserveLocalVariables: true)));
+
+
+            diffError.EmitResult.Diagnostics.Verify(
+#if NETFRAMEWORK
```

Comment:
It'd be better if the test did not depend on the TFM the unit test assembly is built against. You should be able  to pass a specific TargetFramework to one of the CreateCompilation overloads.